### DISCUSSION TITLE
Core Mantainers - Are you in?

### DISCUSSION THREAD

**Discussion by leaofelipe** [clappr/clappr#2122] [General]
Hey, folks! 👋🏻 

In the past few weeks, we've been tidying up our organization to improve our communication and figure out who we can rely on for the next steps and the evolution of Clappr.

I'm starting this discussion to see who still wants to be part of the Clappr Core maintainers and help out with our day-to-day operations.

I'm tagging the @clappr/core team, so if you're interested in continuing, please reply to this thread.

**Comment by paulocesarjr** [comment]
I'm in @paulocesarjr !

**Comment by wijloc** [comment]
Count me in!

**Reply by gugaio** [reply]
Me too @gugaio!!!

**Comment by arielfurtado** [comment]
I'm in!

**Comment by thiagopnts** [comment]
I can help too

**Comment by brunacm-dev** [comment]
I'm in!!!

**Comment by jhonatangcavalcanti** [comment]
Yes, I'm in!
